docker-compose-mastodon-update.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+git stash
+git pull
```

Comment:
I see. Hmm. I'm not sure we'd want to merge this as-is. We recently [updated the upgrade instructions](https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Docker-Guide.md#updating) to explicit tell admins to check a tag rather than the latest. Always running the latest is bold! But I'm not sure we should recommend that generally for admins.